content/en/docs/reference/glossary/ResourceClaim.md

```diff
@@ -0,0 +1,20 @@
+---
+title: ResourceClaim
+id: ResourceClaim
+date: 2023-10-16
+full_link: /docs/concepts/scheduling-eviction/dynamic-resource-allocation/
+short_description: >
+  Defines what kind of resource is needed and what the parameters for it are.
```

Comment:
Most readers won't find this short description helpful. I recommend omitting the glossary entry.